source/WebConfig/cosa_webconfig_internal.c

```diff
@@ -316,6 +319,40 @@ int getWebConfigParameterValues(char **parameterNames, int paramCount, int *val_
 				                k++;
 	     				}
 			      }
+                            else if((strcmp(parameterNames[i], WEBCONFIG_PARAM_SUPPORTED_DOCS) == 0) && (RFC_ENABLE == true))
+                            {
+                                  WebcfgDebug("B4 Get_Webconfig_Supported_Docs\n");
+                                  char* docvalue =  getsupportedDocs();
+                                  paramVal[k]->parameterName = strndup(WEBCONFIG_PARAM_SUPPORTED_DOCS, MAX_PARAMETERNAME_LEN);
+                                  if( (docvalue != NULL) && strlen(docvalue) >0 )
+                                  {
+                                      WebcfgDebug("Webpa get : Datafetched %s\n", docvalue);
+                                      paramVal[k]->parameterValue = strdup(docvalue);
```

Comment:
Would suggest to make either strdup or strndup to make it uniform, as strdup is being used in if case and strndup in else case. Check all cases.